tests/baselines/reference/callWithSpread2.errors.txt

```diff
@@ -36,13 +38,13 @@ tests/cases/conformance/expressions/functionCalls/callWithSpread2.ts(38,1): erro
     rest("d", ...ns)
     
     
-    // this covers the arguments case
+    // extra arguments
     normal("g", ...ns)
-    normal("h", ...mixed)

```

Comment:
Should these `mixed` and `tuple` cases not still be somewhere? (If only to assert they behave similarly)

Reply:
No, correctness is determined syntactically, so the types of the spread arguments don't matter.

Reply:
Alright. :+1: